src/components/App/index.tsx

```diff
@@ -65,25 +62,19 @@ const MapboxLogo = styled.a`
 const App: FC = () => {
   const overlay = useStoreState('overlay');
   const isNavOpen = useStoreState('isNavOpen');
+  const mapHasLoaded = useStoreState('mapHasLoaded');
 
-  const { data: communityData } = useCommunityData();
-  const { data: rainGeoJson } = useRainGeoJson();
-  const { data: pumpsGeoJson } = usePumpsGeoJson();
   const { pathname } = useLocation();
 
   const isHome = pathname === '/';
   const showOverlay = isHome && overlay;
-  const showMap = Boolean(communityData && rainGeoJson && pumpsGeoJson);
```

Comment:
Here was that unnecessary check for the community, adopted, and watered data. In the map component we render these layers only once the data has loaded anyway.